README.md

```diff
@@ -151,29 +151,30 @@ To release all resources you may call `HCaptcha.reset()`.
+| `reportapi`                   | String (URL)            | No       | -                                | See Enterprise docs.                                                                                                                                                 |
+| `assethost`                   | String (URL)            | No       | -                                | See Enterprise docs.                                                                                                                                                 |
+| `imghost`                     | String (URL)            | No       | -                                | See Enterprise docs.                                                                                                                                                 |
+| `customTheme`                 | Stringified JSON        | No       | -                                | See Enterprise docs.                                                                                                                                                 |
+| `host`                        | String (URL)            | No       | -                                | See Enterprise docs.                                                                                                                                                 |
+| `loading`                     | Boolean                 | No       | True                             | Show or hide the loading dialog.                                                                                                                                     |
+| `hideDialog`                  | Boolean                 | No       | False                            | To be used in combination with a passive sitekey when no user interaction is required. See Enterprise docs.                                                          |
+| `tokenExpiration`             | long                    | No       | 120                              | hCaptcha token expiration timeout (seconds).                                                                                                                         |
+| `diagnosticLog`               | Boolean                 | No       | False                            | Emit detailed console logs for debugging                                                                                                                             |
+| `disableHardwareAcceleration` | Boolean                 | No       | True                             | Disable WebView hardware acceleration                                                                                                                                |
```

Comment:
Please bump version and add a changelog.

Reply:
Done